event-bus/api/src/main/java/org/apache/james/events/EventBus.java

```diff
@@ -100,4 +100,8 @@ default Registration register(EventListener.ReactiveGroupEventListener groupList
     default Collection<Group> listRegisteredGroups() {
         return ImmutableList.of();
     }
+
+    void start();
+
+    void restart();
```

Comment:
```suggestion
    default void start() {
    
    }

    default void restart() {
    
    }
```

And drop the other implems?